apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1663,6 +1664,9 @@ static void tc_driver_mtd_ftl_ops(void)
 	fd = open(MTD_FTL_PATH, O_RDWR);
 	TC_ASSERT_GEQ_CLEANUP("open", fd, 0, free(buf));
 
+	ret = ioctl(fd, BIOC_XIPBASE, (unsigned long)&size);
```

Comment:
you should add this to tc_bch instead of here.
Then it will cover more than here.(it will call bch_ioctl -> ftl_ioctl -> part_ioctl -> imxrt_ioctl)

Reply:
Okay, I'll try as you suggested. But this is a temp PR